### DISCUSSION TITLE
Install does not work

### DISCUSSION THREAD

**Discussion by mouragio** [tesla-android/issue-tracker#287] [Q&A] [answered]
Hello all,

I am trying to install teslaandroid 2023.20.1 on RPI 4, 4GB.

SD card is 64GB. I hooked the RPI up to a monitor (used several). After the standard colour screen from RPI; you get a bootloader from Uboot. then starting kernel...

the screen blanks; and no signal HDMI screen is received from the RPI. The RPI is busy as I can tell by the flashing lifgt; but I can wait 5 min or a few hours, I dont get any wifi signal; nor I can tell if the install is still going on.

Any hints, tips and tricks on this one?

thanx

**Comment by mikegapinski** [comment, marked as answer]
Do not use an hdmi display. Tesla Android is headless

**Comment by mouragio** [comment]
Thanx.
All sorted. Install took forever but the issue was a very slow SD card.
bought a fast one and install was fast and its working.
